src/view/cards/Certification/__tests__/Certification.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { render, screen } from 'utils/testing';
+import { mockCerts } from 'services/api/__mocks__';
+import { CertificationCard } from '../Certification';
+
+describe('<Certificates />', () => {
+  const { data: CertificateData } = mockCerts;
```

Comment:
In general, capitalizing the first letter of things should be reserved for classes/interfaces/types. In this case it's just a variable, so we should use camelCase.

```suggestion
  const { data: certificateData } = mockCerts;
```